test-case-reporting/test/fixtures/index.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {KarmaReporter} from 'test-case-reporting';
+import fs from 'fs';
+import path from 'path';
+
+export function getFixtureAsString(name: string): string {
+  return fs.readFileSync(path.join(__dirname, `${name}.json`)).toString('utf8');
+}
+
+/**
+ * Get a JSON test fixture object.
+ */
+export function getFixture(name: string): KarmaReporter.TestResultReport {
```

Comment:
```suggestion
export function getFixture(name: string): Record<string, unknown> {
```

Let's make this `getFixture` helper as generic as possible, since it could conceivably return any JSON. We can always cast it to be more specific at the call site.

Reply:
Done.